packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -0,0 +1,224 @@
+| [Apps for Good](https://www.okta.com/integrations/?category=apps-for-good) | Pre-built integrations with leading apps that enable your workforce to give back |
+
+## Test account guidelines
+
+You need to create a test account on your app so that the OIN team can use it to test and verify your integration. The test account details and credentials are specified  in the **General Settings** > **Test Account** fields from the OIN Manager.
+
+??? Work on:
+
+We need a test user with admin access who has the rights to configure SSO for SAML/SCIM. Even if your support team has configured SSO or SCIM, we need access to verify if users and/or groups on the application end are created by SCIM Provisioning or SAML/OIDC (JiT).
+
+In general, the test account allows the OIN team to verify that your integration flow works as expected for your use case.
+
+We prefer isvtest@okta.com to be a test user. If you can't use this email to create a test user, you can provide a specific account in a different domain.
+
+??? Perhaps remove/merge with the above ???
+
+Depending on the use case of our app, the permissions and roles for your test account are varied. For example:
+
+* For an SSO integration, you can provide a regular, non-admin user test account for the OIN team to test Single Sign-On with your app.
+* For a lifecycle management integration, you can provide a test account with HR admin privileges to onboard, change roles, or offboard employees on your app.
+* For an API service integration, you can provide a test account with privileges to trigger an API process on your app.
+
+> **Note:** Ensure that proper scopes are defined for OAuth 2.0 authentication.
+
+In general, the test account provides the OIN team with the ability to verify that your integration flow works as expected for your use case. Provide the necessary permissions required for your app. If you have further questions, contact the OIN team.
+
+## Customer configuration document guidelines
+
+A configuration guide helps your customers understand how to configure your Okta integration to work with your cloud application.
+
+You need to provide a configuration guide as part of the OIN submission process. Your guide is provided to administrators through the Okta Admin Console. Okta checks your document for general adherence to the configuration guide instructions. After your integration is published to the OIN, you can make the link public or customer‐accessible.
+
+>**Note:** A configuration guide is required for SCIM and OIDC integrations. It's optional for SAML integrations, as Okta supplies its own documentation with the apps.
+
+You can create the guide in whatever format works best for you: a webpage, a Google or Word doc, or a PDF are all acceptable.
```

Comment:
@mohammadabdullah-okta , wouldn't it be best to collect the actual link so that you don't need to email and ask for that later?

I wonder if we should be collecting a draft link (optional), published link (required)

Reply:
+1 on link so that there's consistency and reduces back and forth with the ISV

Reply:
I'll update this sentence to request for a doc link that is publicly accessible